src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <div>
+    <div>
+      <label>{{ selectLabel }} <span v-if=isRequired aria-label='Required'>*</span></label>
+    </div>
+    <div 
+      :aria-expanded="opened"
+      :aria-owns="id"
+      :aria-label="selectLabel"
+      role="combobox"
+      tabindex="0"
+      v-model="selectLabel"
```

Comment:
Keyboard navigation is needed. This div needs a keypress handler so that it opens on Space. Focus then needs to go to the selected option in the list (or if none, the first option) and then key up/key down on the li elements should move focus. Enter should change selection and close the select box.